build.gradle

```diff
@@ -142,19 +143,56 @@ tasks.withType(JavaCompile).configureEach {
 	options.compilerArgs.add '-XDstringConcat=inline'
 }
 
-tasks.named("build").configure { dependsOn('installLocalGitHookPostCheckout') }
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-	group = 'git'
-	from new File(rootProject.rootDir, '.scripts/post-checkout')
-	into { new File(rootProject.rootDir, '.git/hooks') }
-	filePermissions {
-		unix(0775)
+// Get a path as relative to the root directory of the project (usually for printing)
+def getRootRelativePath(path) {
+	rootDir.toPath().relativize(path)
+}
+
+// Setup local git hooks
+def ensureSymlink(link, target) {
```

Comment:
maybe change func name to represent the comment instead of adding comment

Reply:
the comment is for that section of code, not just the function. I'll make that more obvious